Vagrantfile

```diff
@@ -78,6 +78,9 @@ Vagrant.configure("2") do |config|
     centos.vm.provision "common", type: "shell", inline: $bootstrap_rhel_common
     centos.vm.provision "shell", inline: <<-SHELL
        dnf install -y centos-release-nfv-openvswitch
+       sed -i s/mirror.centos.org/vault.centos.org/g /etc/yum.repos.d/CentOS-NFV-OpenvSwitch.repo
+       sed -i s/^#.*baseurl=http/baseurl=http/g /etc/yum.repos.d/CentOS-NFV-OpenvSwitch.repo
+       sed -i s/^mirrorlist=http/#mirrorlist=http/g /etc/yum.repos.d/CentOS-NFV-OpenvSwitch.repo
```

Comment:
The same thing is done just above in the file. Is that needed because `dnf install -y centos-release-nfv-openvswitch` is not performed early enough?

The two should be consolidated, IMO moving the mirror fixup from above here.

Reply:
So,  in order to be able to do:
```
 dnf install -y centos-release-nfv-openvswitch
```
, first, the repository must be changed from mirror. ... to vault. ... in all exsiting .repo files otherwise, that RPM can't be installed.

But, the RPM centos-release-nfv-openvswitch then installs /etc/yum.repos.d/CentOS-NFV-OpenvSwitch.repo with mirror. ... and so the file generated by that RPM must be fixed, as well.

There's really no way to consolidate it?

Reply:
done